x-pack/test/functional/apps/timelion/feature_controls/timelion_security.ts

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from 'expect.js';
+import { KibanaFunctionalTestDefaultProviders } from 'x-pack/test/types/providers';
+
+// tslint:disable no-default-export
+export default function({ getPageObjects, getService }: KibanaFunctionalTestDefaultProviders) {
+  const esArchiver = getService('esArchiver');
+  const security = getService('security');
+  const PageObjects = getPageObjects(['common', 'timelion', 'header', 'security', 'spaceSelector']);
+  const find = getService('find');
+  const appsMenu = getService('appsMenu');
+
+  describe('feature controls security', () => {
+    before(async () => {
+      await esArchiver.loadIfNeeded('timelion/feature_controls');
```

Comment:
Any reason we shouldn't unload this in the after?

Reply:
I didn't only because it seemed like we don't unload the archives that are loaded "ifNeeded" in other places

Reply:
Ah, missed that difference.